janitor/functions/pivot.py

```diff
@@ -190,13 +218,15 @@ def pivot_wider(
     Reshapes data from *long* to *wide* form.
 
     The number of columns are increased, while decreasing
-    the number of rows. It is the inverse of the `pivot_longer`
+    the number of rows. It is the inverse of the
+    [`pivot_longer`][janitor.functions.pivot_longer.pivot_longer]
```

Comment:
This should probably be `janitor.functions.pivot.pivot_longer`, I think.

Reply:
I followed the example you suggested

Reply:
Well, I didn't mean for you to copy it literally :) 
As I understand, the path in the square brackets is the path to the function itself. `pivot_longer` lives inside a file called `pivot`, not `pivot_longer`. That's why it's not rendering properly at the moment.

![image](https://user-images.githubusercontent.com/30731072/152677996-0a69ae8c-d1ca-4dc0-967e-b35644715638.png)

Reply:
😀

Reply:
fixed